README.md

```diff
@@ -1,20 +1,122 @@
 
-# CLI support for Responsible AI
+This new feature offers users a new powerful and robust toolkit for understanding your model and data in order to develop your machine learning models responsibly, now all in one place and integrated with your AzureML workspace.
 
-## Create a Model Analysis using the CLI
-Responsible AI job creation in the CLI is based on .YAML config files that you as a user can create. We have provided samples of these .YAML files that you can update with your specific settings. Follow this quickstart guide to create Responsible AI jobs with the CLI
-- [How to create an Responsible AI job with the CLI](https://github.com/RachelKellam/RAI-vNext-Preview/blob/main/docs/cli/how-to-create-model-analysis-cli.md)
+❗ **Please note:** This initial version of the Responsible AI dashboard currently does not support the integration of fairness metrics. For fairness metrics, please refer to our existing offering [here.](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-machine-learning-fairness-aml)
 
-# Python SDK Support for Responsible AI
-- [How to create an Responsible AI job with the Python SDK](https://github.com/Azure/RAI-vNext-Preview/blob/main/docs/PythonSDK/how-to-create-model-analysis-python.md)
+## Supported scenarios, models and datasets
+
+`azureml-responsibleai` supports computation of Responsible AI insights for `scikit-learn` models that are trained on `pandas.DataFrame`. The `azureml-responsibleai` accept both models and pipelines as input as long as the model or pipeline implements a `predict` or `predict_proba` function that conforms to the `scikit-learn` convention. If not compatible, you can wrap your model's prediction function into a wrapper class that transforms the output into the format that is supported (`predict` or `predict_proba` of `scikit-learn`), and pass that wrapper class to modules in `azureml-responsibleai`.
+
+Currently, we support datasets having numerical and categorical features. The following table provides the scenarios supported for each of the four responsible AI insights:-
+
+| RAI insight | Binary classification | Multi-class classification | Multilabel classification | Regression | Timeseries forecasting | Categorical features | Text features | Image Features | Recommender Systems | Reinforcement Learning |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | -- |
+| Explainability | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+| Error Analysis | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+| Causal Analysis | Yes | No | No | Yes | No | Yes (max 5 features due to expensiveness) | No | No | No | No |
```

Comment:
I'm not keen on 'expensiveness' - "computational cost" feels a bit better to me.